chevah_build

```diff
@@ -203,6 +204,16 @@ case $OS in
         export CFLAGS="$CFLAGS -mmacosx-version-min=10.8"
         export MACOSX_DEPLOYMENT_TARGET=10.8
         ;;
+    macos*)
+        # On macOS 10.12 or newer we need the Homebrew version of OpenSSL,
+        # as Apple only provides an old OpenSSL version: 0.9.8.
+        export CC="clang"
+        export CXX="clang++"
```

Comment:
It's true `$CXX` is not really needed, but it helps to define it, especially when debugging the build. However, `$CPPFLAGS` is needed. This is documented above when `CXX` is defined for the first time:
```
# CXX is not really needed, we export it to make sure g++ won't get picked up
# when not using gcc, and thus silence the associated configure warning.
# However, we set $CPPFLAGS later for linking to statically-compiled libs.
export CXX='g++'
```